samples/openthread/coprocessor/README.rst

```diff
@@ -70,49 +70,50 @@ The sample supports the following development kits for testing the network statu
    :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf21540dk_nrf52840
 
 To test the sample, you need at least one development kit.
-Additional development kits programmed with the NCP sample can be used for the :ref:`optional testing of network joining <ot_ncp_sample_testing_more_boards>`.
+Additional development kits programmed with the Co-Processor sample can be used for the :ref:`optional testing of network joining <ot_coprocessor_sample_testing_more_boards>`.
 
 Moreover, the sample requires a Userspace higher layer process running on user's device in order to communicate with the MCU NCP part.
-This sample uses `wpantund`_ as reference.
+This sample uses `wpantund`_ as reference for NCP variant and :ref:`ug_thread_tools_ot_daemon` as reference for RCP variant.
 
 User interface
 **************
 
 All the interactions with the application are handled using serial communication.
 
-For the interaction with the application, this sample uses :ref:`ug_thread_tools_wpantund` process with ``wpanctl`` commands.
+For the interaction with the application, the NCP variant of this sample uses :ref:`ug_thread_tools_wpantund` process with ``wpanctl`` commands and the RCP variant uses :ref:`ug_thread_tools_ot_daemon` with commands listed in `OpenThread CLI Reference`_.
```

Comment:
Not entirely true.
The RCP can be also used together with the `ot-ncp` host app and interact with the wpantund as well.
https://github.com/openthread/openthread/blob/master/src/posix/README.md#with-real-device-1